### ISSUE TITLE
[FEATURE] x env upgrade | [mod=env] [v0.4.15]

### ISSUE THREAD

**Issue by iandol** [x-cmd/x-cmd#107]
**Describe the feature**

`x env` installs packages, but how can we upgrade the packages to the latest versions? `x env upgrade` would upgrade all packages to the newest versions. at present we need to go through each install manually and use a TUi to get the name then run use and select (1), quite tedious.

**Is is urgent**

Low priority but would be nice to have. I wanted to try to list all installed packages then run `x env use` but the `ls` command always uses TUI so can't get the list of installed packages as text so can't then automate it. `x env use existingpackage` also brings up a TUI so cannot automate it...

**Additional context**

**Comment by edwinjhlee** [comment]
TO DO:

```bash
x env upgrade <pkg> == x env use --upgrade <pkg>
x env upgrade --all == x env use --upgrade --all
```

**Comment by curry798** [comment]
The functionality has been implemented in v0.4.15

TLDR:
```
x env upgrade jq yq
x env upgrade --all
```

This issue is being closed due to inactivity.  
Please reopen it if you encounter further problems.

**Comment by iandol** [comment]
Thanks!!! Is there a way to stop interactive questions like "Do you want to recycle pkg ..."? Something like `x env upgrade --all --force`...

**Comment by curry798** [comment]
I am closing this issue ~ If problems occured, feel free to reopen.

Thank you. @iandol
